### PR TITLE
feat(session): Add Local Weather Options

### PR DESCRIPTION
**Pull request by xM4ddy**

### CODE REVIEW

**Comment by xM4ddy** [comment]
Hmm some weird commit shit happened but still looks right.

File: BigBaseV2/src/views/view_session.cpp [hunk 1]
```diff
@@ -64,6 +65,24 @@ namespace big
...
+				QUEUE_JOB_BEGIN_CLAUSE()
+				{
+					MISC::CLEAR_OVERRIDE_WEATHER();
+				}
+				QUEUE_JOB_END_CLAUSE
+			}
+
+			if (ImGui::ListBox("", &g->session.local_weather, session::weathers, 15))
+			{
+				session::local_weather();
```
Comment (LiamD-Flop) [inline BigBaseV2/src/views/view_session.cpp]: As you're using a native call this in a fiber_pool

**Review by LiamD-Flop — CHANGES_REQUESTED** [review]
- [ ] Don't call natives in GUI thread

File: BigBaseV2/src/views/view_session.cpp (hunk 1 above)
Comment (Yimura) [reply BigBaseV2/src/views/view_session.cpp]: Yes just the session_local weather call since that function uses a native.

**Review by LiamD-Flop — APPROVED** [review]
Weather be looking GOOD

**Pull request merged**